### PR TITLE
install process & bump lattice-sagas, lattice-ui-kit, and lattice-utils

### PR DESCRIPTION
**Pull request by bergieboy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,14 +44,15 @@
     "jspdf": "2.3.1",
     "lattice": "~0.89.2",
     "lattice-auth": "0.22.0",
-    "lattice-sagas": "~0.63.0",
-    "lattice-ui-kit": "~0.43.1",
-    "lattice-utils": "~0.10.0",
+    "lattice-sagas": "~0.66.0",
```
Comment (UnsungHero97) [inline package.json:47]: Caution - `lattice` and `lattice-sagas` go hand in hand, so it's a bit dangerous to bump only `lattice-sagas`. What are you looking for in `v0.66.0`? Any reason not to bump `lattice` as well?

Comment (bergieboy) [reply package.json:47]: yeah, I'm looking for anything new in `lattice-sagas`, `lattice`, or `lattice-utils` just taking this as an opportunity to bump them. this was an oversight.

File: package.json [hunk 2]
```diff
@@ -42,16 +42,17 @@
     "history": "~4.10.0",
     "immutable": "4.0.0-rc.9",
     "jspdf": "2.3.1",
-    "lattice": "~0.89.2",
+    "lattice": "~0.92.0",
```
Comment (UnsungHero97) [inline package.json:45]:
```suggestion
    "lattice": "~0.93.4",
```

File: package.json (hunk 1 above)
Comment (UnsungHero97) [reply package.json:47]:
```suggestion
    "lattice-sagas": "~0.67.3",
```

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -42,16 +42,17 @@
...
     "lattice-auth": "0.22.0",
-    "lattice-sagas": "~0.63.0",
-    "lattice-ui-kit": "~0.43.1",
-    "lattice-utils": "~0.10.0",
+    "lattice-sagas": "~0.66.0",
```
Comment (UnsungHero97) [inline package.json:47]:
```suggestion
    "lattice-sagas": "~0.67.3",
```

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

**Review by UnsungHero97 — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -42,16 +42,17 @@
...
     "immutable": "4.0.0-rc.9",
     "jspdf": "2.3.1",
-    "lattice": "~0.89.2",
+    "lattice": "~0.93.4",
     "lattice-auth": "0.22.0",
-    "lattice-sagas": "~0.63.0",
-    "lattice-ui-kit": "~0.43.1",
-    "lattice-utils": "~0.10.0",
+    "lattice-sagas": "~0.67.3",
+    "lattice-ui-kit": "~0.45.0",
```
Comment (solomontang) [inline package.json:48]: 0.45.1 is out but doesn't really matter for this repo

**Review by solomontang — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Pull request merged**